app/webpacker/components/RegistrationsV2/Registrations/RegistrationList.jsx

```diff
@@ -95,7 +96,7 @@ export default function RegistrationList({ competitionInfo }) {
   const changeSortColumn = (name) => dispatch({ type: 'CHANGE_SORT', sortColumn: name });
 
   const [psychSheetEvent, setPsychSheetEvent] = useState();
-  const [psychSheetSortBy, setPsychSheetSortBy] = useState('single');
+  const psychSheetSortBy = useMemo(() => (psychSheetEvent === '333mbf' ? 'single' : 'average'), [psychSheetEvent]);
```

Comment:
IIRC in the old system all blindfolded events were sorted by single, not just `333mbf`